apps/stories/api_views.py

```diff
@@ -905,12 +938,23 @@ def get_json(self, source, stories_qset, school_counts):
             old_id_key = 'school__admin3__parent__parent__id'
             old_name_key = 'school__admin3__parent__parent__name'
 
+            # json['gka_districts'] = [
+            #     {
+            #         'id': item[old_id_key],
+            #         'name': item[old_name_key]
+            #     }
+            #     for item in gka_districts_queryset
+            # ]
+
             json['gka_districts'] = [
                 {
-                    'id': item[old_id_key],
-                    'name': item[old_name_key]
+                    'id':15700,
+                    'name':'rayagada'
+                },
+                {
+                    'id':11725,
+                    'name':'bolangir'
```

Comment:
We were calculating the GKA districts from the SMSes received. Since we don't have any SMSes for Odisha to begin with, that fails. This is hardcoded for now, but should ideally be replaced with something like:
```
programme = Programme.objects.get(name='Ganitha Kanika Andolana')
         gka_school_q = school_qset.filter(programmes=programme)
         gka_school_q = school_qset.filter(programmes=programme)
+        gka_districts = gka_school_q.values_list(
+            'admin3__parent__parent',
+            flat=True
+        ).distinct(
+            'admin3__parent__parent'
+        )
```